dashboard/test/ui/features/acquisition_products/hoc_guide_dialog.feature

```diff
@@ -0,0 +1,32 @@
+Feature: HOC Guide Email Conversion Dialog
+
+  @eyes
+  Scenario: View and Close Dialog as signed in Instructor
+    When I open my eyes to test "signedIn"
```

Comment:
should this have a more descriptive name? I can't remember how this shows up on Applitools (may not be necessary to rename if the context is clear)

Reply:
Good call- updated!